OrbitService/ProcessList.cpp

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "ProcessList.h"
+
+#include <absl/strings/ascii.h>
+#include <absl/strings/numbers.h>
+#include <absl/strings/str_format.h>
+
+#include <filesystem>
+#include <unordered_map>
+
+#include "LinuxUtils.h"
+#include "OrbitBase/Logging.h"
+
+namespace {
+
+std::string ReadFile(const std::string& filename) {
+  std::stringstream buffer;
+  std::ifstream in_file(filename);
+  if (in_file.fail()) {
+    ERROR("Unable to read contents of file \"%s\"", filename.c_str());
```

Comment:
outcome::failure?

Reply:
It is actually not possible to have `outcome::result<std::string, std::string>`. It cant be the same type for return type and error type. I just realized there is a `FileToString()` method in OrbitCore/Utils.h. I am using that one for now, and will do another PR which implements an Error class for OrbitCore and then use that.

Reply:
Yeah you will need a wrapper struct something like struct Error { std::string error; };